scripts/phpspy_build.sh

```diff
@@ -5,6 +5,16 @@
 #
 set -euo pipefail
 
+# TODO support aarch64
+if [ $(uname -m) != "x86_64" ]; then
+    mkdir -p /phpspy
+    touch /phpspy/phpspy
+    mkdir -p /binutils/binutils-2.25/bin/bin/
+    touch /binutils/binutils-2.25/bin/bin/objdump
+    touch /binutils/binutils-2.25/bin/bin/strings
+    exit 0
```

Comment:
kinda ugly we package empty file as resources don't you think ?

Reply:
Yup, but it seems that there's no true, real way to have conditional `COPY`

Reply:
How about that in the builder containers you install everything into `/installations/`, and when you copy from the builder container to the gProfiler container you just copy the `/installations/*` dir into the dest dir. That way the dir could be empty instead of having empty files

Reply:
I thought about it. But
1. `COPY` [still requires 1 valid target](https://stackoverflow.com/a/46801962/797390), so even in the "no files" case needs 1 dummy file.
2. This will allow implicitly "failing" to copy the file in cases we expect it to exist.

I know this sucks but I suppose that's the least worse option here